public/locales/fr-FR/translations.json

```diff
@@ -0,0 +1,477 @@
+{
+  "action": "action",
+  "assets": "Actifs",
+  "assets.nft_tab_title": "Jetons non fongibles",
```

Comment:
Should each of these words be capitalized since this is a title? 

(More general pattern - looking at the english translation, some of these phrases use Title Case, while others use all lowercase, and others use special capitalization. I think it'd be worth doing a check through to make sure the translated match the casing - I think most do from the look of it, but since I don't speak French I can't tell what words shouldn't be capitalized)

https://github.com/ripple/explorer/blob/staging/public/locales/en-US/translations.json